packages/@uppy/compressor/src/index.js

```diff
@@ -66,6 +67,10 @@ export default class Compressor extends BasePlugin {
         message: this.i18n('compressingImages'),
       })
 
+      if (file.isRemote) {
+        return Promise.resolve()
```

Comment:
I think it would be nice to add a comment explaining why we need to shortcut here.